src/coffee/client.coffee

```diff
@@ -1,6 +1,7 @@
 _ = require 'underscore'
 Rest = require './connect/rest'
 TaskQueue = require './task-queue'
+AuthenticationService    = require './services/authentication'
```

Comment:
I would name it `CustomerAuthenticationService`, otherwise it's a bit misleading